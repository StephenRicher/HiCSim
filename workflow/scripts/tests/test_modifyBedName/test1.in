# header
browser
track
1	1	100	name1	600	.	124:191
1	10	100	name2	1000	.	124:191
6	90	100	name3	20	.	124:191
chr11	99	100	name4	10	.	124:191
5	10	100	name5	500	.	124:191
4	1	150	name6	550	.	124:191
1	5000	6000	name7	400	.	124:191
1	1	50	name8	300	.	124:191
1	25	30	name9	250	.	124:191
1	75	90	name10	0	.	124:191
