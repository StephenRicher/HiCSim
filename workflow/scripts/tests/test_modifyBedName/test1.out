1	1	100	1:1-100:name1:600	600
1	10	100	1:10-100:name2:1000	1000
6	90	100	6:90-100:name3:20	20
chr11	99	100	chr11:99-100:name4:10	10
5	10	100	5:10-100:name5:500	500
4	1	150	4:1-150:name6:550	550
1	5000	6000	1:5000-6000:name7:400	400
1	1	50	1:1-50:name8:300	300
1	25	30	1:25-30:name9:250	250
1	75	90	1:75-90:name10:0	0
